public/styles.css

```diff
@@ -230,11 +229,87 @@ header {
   font-size: 110%;
   font-weight: bold;
 }
-meter {
-  background: grey;
-  -ms-progress-appearance: ring;
+meter::-webkit-meter-bar {
+  background: none;
+  background: lightgrey;
+}
+meter::-webkit-meter-optimum-value {
+  background-image: linear-gradient(90deg, #5c5c5c 50%, #5c5c5c 100%);
+}
+#myModal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0,0,0);
+  background-color: rgba(0,0,0,0.4);
+}
+#myModal-content { /* this is the only way to access my modal */
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  gap: 10px;
+  background-color: #fefefe;
+  margin: 15% auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%;
+}
+
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+.close:hover,
+.close:focus {
+  color: black;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+#radioTable {
+}
+#radioTable2 {
+  display: table;
+  width: 65vw;
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+
+td { /*this may cause issues but I can go back later and fix */
+  border: 1px solid black;
+  text-align: center;
```

Comment:
what issues do you see this causing?

Reply:
I did a lot of CSS changes and had a prior merge conflict with the QA section. It is possible there is a conflict but unlikely.

Reply:
What merge conflict were you experiencing?

Reply:
There was an issue with some overlap in the CSS file between Jacob's modal and mine. Merge conflicts should be resolved now.

Reply:
Got it. Thanks for clarifying!